### PR TITLE
Release 2.8.0

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming v2.8.0 production release? Well you've just found it, congratulations! :tada:

This is based on the development.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

**Comment by tidy-dev** [comment]
Feature Flags as expected:
```
/** Should the app show hide whitespace in changes tab */
export function enableHideWhitespaceInDiffOption(): boolean {
  return true
}

/** Should we allow expanding text diffs? */
export function enableTextDiffExpansion(): boolean {
  return true
}

/** Should we allow using the save dialog when choosing where to clone a repo */
export function enableSaveDialogOnCloneRepository(): boolean {
  return true
}

/** Should we allow setting repository aliases? */
export function enableRepositoryAliases(): boolean {
  return true
}
```

All other feature flags are either false or beta, except trampoline which is true from last prod release.

**Comment by tidy-dev** [comment]
~New 'architecture' metric to be deployed to central. https://github.com/github/central/pull/529~
~Needs added to website.~

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.7.3": [
+      "[New] Expand diffs to get more context about your changes - #7014",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Expand diffs to view more context around your changes - #7014",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+      "[New] Create aliases for repositories you want to be displayed differently in the repository list - #7856",
+      "[Added] Hide whitespace in diffs on Changes tab - #6818. Thanks @say25!",
+      "[Fixed] Show correct state of remote branch when deleting local branches - #11923",
+      "[Fixed] On hover, show the full branch name in tooltip - #12008",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Display the full branch name in a tooltip when hovering over a branch in the branches list - #12008",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+      "[Improved] During cloning progress and on hover, show full progress description - #11375. Thanks @maifeeulasad!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Display full progress description when cloning a repository - #11375. Thanks @maifeeulasad!",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+      "[Improved] Cherry-pick multiple non sequential commits - #12030",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Cherry-pick multiple non-sequential commits - #12030",
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
This is looking great! A few suggestions, but all very minor and non-substantive.

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.7.3": [
```
Comment (sergiou87) [inline changelog.json]: Shouldn't this be 2.8.0?
```suggestion
    "2.8.0": [
```

**Review by sergiou87 — CHANGES_REQUESTED** [review]

**Review by billygriffin — APPROVED** [review]
🚀

**Review by sergiou87 — APPROVED** [review]
:shipit: 🚀

**Comment by wahabzaib555** [comment]
> ## Description
> Looking for the PR for the upcoming v2.8.0 production release? Well you've just found it, congratulations! 🎉
> 
> This is based on the development.
> 
> ## Release checklist
> * [x]  Check to see if there are any errors in Sentry that have only occurred since the last production release
> * [x]  Verify that all feature flags are flipped appropriately
> * [x]  If there are any new metrics, ensure that central and desktop.github.com have been updated

**Pull request merged**